### PR TITLE
Fix/bug

### PR DESCRIPTION
**Pull request by GooJinSun**
## What does this PR do?

- #261 comment item이 overflow되는 버그 수정
- #258 ' ' white space 빈 값이 post되는 버그 수정
- share settings test 추가
- ## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code Style Update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: frontend/src/components/comments/CommentItem.jsx [hunk 1]
```diff
@@ -31,7 +31,8 @@ const CommentItemWrapper = styled.div`
 CommentItemWrapper.displayName = 'CommentItem';
 
 const CommentContent = styled.div`
-  margin: 0 8px;
+  margin: 4px 8px 0 8px;
+  word-break: break-all;
```
Comment (koyrkr) [inline frontend/src/components/comments/CommentItem.jsx:35]: 우와 이런것두 있구나.. 강제로 줄바꿈을 해주는거네 신기방기

File: frontend/src/components/posts/ShareSettings.jsx [hunk 2]
```diff
@@ -171,6 +184,29 @@ export default function ShareSettings({
...
+          <Button
+            id="confirm-button"
+            onClick={() => setNoContentDialogOpen(false)}
+            color="primary"
+            autoFocus
+          >
+            확인
+          </Button>
+        </DialogActions>
+      </Dialog>
```
Comment (koyrkr) [inline frontend/src/components/posts/ShareSettings.jsx:209]: 오오오 좋아요!!!! 👍 👍

**Review by koyrkr — APPROVED** [review]
버그킬러~~~~

**Comment by ssm0318** [comment]
에프킬라인줄~

**Review by curious-curie — APPROVED** [review]

**Pull request merged**